pages/snapshots/[id].js

```diff
@@ -32,37 +33,58 @@ const SnapshotSummary = ({ resources, initialSnapshot, token }) => {
     snapshot.vulnerabilities = selected.vulnerabilities;
     setHasValue(
       snapshot.assets.length > 0 ||
-        snapshot.vulnerabilities.length > 0 ||
-        snapshot.notes
+      snapshot.vulnerabilities.length > 0 ||
+      snapshot.notes
     );
   };
 
   const updateNotes = notes => {
     snapshot.notes = notes;
     setHasValue(
       snapshot.assets.length > 0 ||
-        snapshot.vulnerabilities.length > 0 ||
-        snapshot.notes
+      snapshot.vulnerabilities.length > 0 ||
+      snapshot.notes
     );
   };
 
-  const { dob, firstName, lastName, assets, vulnerabilities, notes } = snapshot;
-  const customerId = snapshot.systemIds?.[0];
-
+  const { dob, firstName, lastName, postcode, assets, vulnerabilities, notes } = snapshot;
+  let customerId = snapshot.systemIds?.[0];
+  // check the external system id for redirecting back to originating system
+  let backtoSingleView = true;
+  if (customerId && customerId.includes("inh-", 0)) {
+    customerId = customerId.substring(4);
+    backtoSingleView = false;
+  }
+  const residentCoordinates = geoCoordinates(postcode);
+  
   return (
     <>
       <div>
+        { backtoSingleView && (
         <a
           href={`${process.env.NEXT_PUBLIC_SINGLEVIEW_URL}/customers/${customerId}/view`}
           className="govuk-back-link back-button"
           data-testid="back-link-test"
         >
           Back to Single View
         </a>
+        )}
+        
+        { !backtoSingleView && (
+          <a
+          href={`${process.env.INH_URL}/help-requests/edit/${customerId}`}
+          className="govuk-back-link back-button"
+          data-testid="back-link-test"
+        >
+          Back to I Need Help
+        </a>
+
+        )}
```

Comment:
Now we've forked, could we have this as the default rather than any single view options?

(it's OK to address this after the merge)

Reply:
Yes I will fix later